src/webui/www/public/scripts/mocha-init.js

```diff
@@ -451,6 +451,36 @@ initializeWindows = function() {
         }
     };
 
+    copyNameFN = function() {
+        var selectedRows = torrentsTable.selectedRowsIds();
+        var names = [];
+        if (selectedRows.length) {
+            var rows = torrentsTable.getFilteredAndSortedRows();
+            for (var i = 0; i < selectedRows.length; i++) {
+                var hash = selectedRows[i];
+                names.push(rows[hash].full_data.name);
+            }
+        }
+        return names.join("\n");
+    };
+
+    copyMagnetLinkFN = function() {
+        var h = torrentsTable.selectedRowsIds();
```

Comment:
Oh, IMO using the same icon (Copy name) is fine.
Currently on WebUI `Copy hash` icon is the same as `Copy magnet`, this isn't right.

Also, you might want to reorder the copy * order:
1.copy name
2.copy hash
3.copy magnet